include/nlohmann/detail/output/binary_writer.hpp

```diff
@@ -1433,7 +1479,7 @@ class binary_writer
     /*!
     @brief determine the type prefix of container values
     */
-    CharType ubjson_prefix(const BasicJsonType& j) const noexcept
+    CharType ubjson_prefix(const BasicJsonType& j, const bool use_bjdata = false) const noexcept
```

Comment:
Will there ever be a place where this is defaulted, or should it always be provided so that we can't forget it?

Reply:
fixed